deployments/arbitrum.diamond.immutable.json

```diff
@@ -49,9 +49,9 @@
         "Name": "HopFacetOptimized",
         "Version": "2.0.0"
       },
-      "0xA57862295BdF8A680fda893B9bd3b935236d18bA": {
+      "0x58FBa880b973edB5E4f6Aa4e2bD3b15c865b9B18": {
         "Name": "HopFacetPacked",
-        "Version": "1.0.1"
+        "Version": ""
```

Comment:
We don't need to specify the version here?

Reply:
agreed. This should not be happening. Please try to re-run it.